src/backend/marsha/core/api.py

```diff
@@ -393,13 +393,9 @@ def post(self, request):
 
         consumer_site = video.playlist.consumer_site
 
-        lrs_url = consumer_site.lrs_url or getattr(settings, "LRS_URL", "")
-        lrs_auth_token = consumer_site.lrs_auth_token or getattr(
-            settings, "LRS_AUTH_TOKEN", ""
-        )
-        lrs_xapi_version = consumer_site.lrs_xapi_version or getattr(
-            settings, "LRS_XAPI_VERSION", ""
-        )
+        lrs_url = consumer_site.lrs_url
+        lrs_auth_token = consumer_site.lrs_auth_token
+        lrs_xapi_version = consumer_site.lrs_xapi_version
```

Comment:
maybe we don't need these variables anymore then.

Reply:
why not